internal/core/workers/operation_execution_worker/operation_execution_worker.go

```diff
@@ -88,6 +95,11 @@ func (w *OperationExecutionWorker) Start(ctx context.Context) error {
 			if err != nil && shouldFinishWorker(err) {
 				return err
 			}
+		case <-healthControllerTicker.C:
+			err := w.createHealthControllerOperation(w.workerContext)
```

Comment:
Shouldn't we check if there is another health_controller operation on the queue? Looks to me that simply creating a new operation when the ticker rings could lead to a case where we have like 2 health operations enqueued

Reply:
There is no problem if this happen, because in this case the first one could generate a change in the scheduler, otherwise it will only check and finish (because the cluster will be in the desired state)